backend/src/controllers/social.controller.ts

```diff
@@ -16,6 +16,19 @@ export const getAll = async (_req: Request, res: Response) => {
   }
 };
 
+export const getAllActive = async (_req: Request, res: Response) => {
+    try {
+    const socials = await Social.find({active: false});
```

Comment:
Are you sure about this condition? We want to get only active, not inactive ones

Reply:
OK sorry. I forgot to switch flag :)